tools/build-kani/src/sysroot.rs

```diff
@@ -56,35 +56,42 @@ pub fn kani_playback_lib() -> PathBuf {
 }
 
 /// Returns the path to where Kani's pre-compiled binaries are stored.
-pub fn kani_sysroot_bin() -> PathBuf {
+fn kani_sysroot_bin() -> PathBuf {
     path_buf!(kani_sysroot(), "bin")
 }
 
 /// Build the `lib/` folder and `lib-playback/` for the new sysroot.
 /// - The `lib/` folder contains the sysroot for verification.
 /// - The `lib-playback/` folder contains the sysroot used for playback.
-pub fn build_lib() -> Result<()> {
-    build_verification_lib()?;
-    build_playback_lib()
+pub fn build_lib(bin_folder: &Path) -> Result<()> {
+    let compiler = bin_folder.join("kani-compiler");
+    build_verification_lib(&compiler)?;
+    build_playback_lib(&compiler)
 }
 
 /// Build the `lib/` folder for the new sysroot used during verification.
 /// This will include Kani's libraries as well as the standard libraries compiled with --emit-mir.
-fn build_verification_lib() -> Result<()> {
+fn build_verification_lib(compiler: &Path) -> Result<()> {
```

Comment:
Please rename `compiler` to `compiler_path` or any other name that hints at this being a location, not an object.

Reply:
Sure